microsoft-iis-mixin/alerts.libsonnet

```diff
@@ -1,87 +1,87 @@
         rules: [
           {
-            alert: 'MicrosoftIISHighNumberOfRejectedAsyncIORequests',
+            alert: 'MicrosoftIISRejectedAsyncIORequests',
             expr: |||
               increase(windows_iis_rejected_async_io_requests_total[5m]) > %(alertsWarningHighRejectedAsyncIORequests)s
-            ||| % $._config,
+            ||| % this.config,
             'for': '5m',
             labels: {
               severity: 'warning',
             },
             annotations: {
               summary: 'There are a high number of rejected async I/O requests for a site.',
-              description: |||
-                The number of rejected async IO requests is {{ printf "%%.0f" $value }} over the last 5m on {{ $labels.instance }} - {{ $labels.site }} which is above the threshold of %(alertsWarningHighRejectedAsyncIORequests)s.
-              ||| % $._config,
+              description:
+                ('The number of rejected async IO requests is {{ printf "%%.0f" $value }} over the last 5m on {{ $labels.instance }} - {{ $labels.site }}, ' +
+                 'which is above the threshold of %(alertsWarningHighRejectedAsyncIORequests)s.') % this.config,
             },
           },
           {
-            alert: 'MicrosoftIISHighNumberOf5xxRequestErrors',
+            alert: 'MicrosoftIIS5xxRequestErrors',
             expr: |||
-              sum without (pid, status_code)(increase(windows_iis_worker_request_errors_total{status_code=~"5.*"}[5m])) > %(alertsCriticalHigh5xxRequests)s
-            ||| % $._config,
+              sum without (pid, status_code)(increase(windows_iis_worker_request_errors_total{status_code=~"5.."}[5m])) > %(alertsCriticalHigh5xxRequests)s
+            ||| % this.config,
             'for': '5m',
             labels: {
               severity: 'critical',
             },
             annotations: {
               summary: 'There are a high number of 5xx request errors for an application.',
-              description: |||
-                The number of 5xx request errors is {{ printf "%%.0f" $value }} over the last 5m on {{ $labels.instance }} - {{ $labels.app }} which is above the threshold of %(alertsCriticalHigh5xxRequests)s.
-              ||| % $._config,
+              description:
+                ('The number of 5xx request errors is {{ printf "%%.0f" $value }} over the last 5m on {{ $labels.instance }} - {{ $labels.app }}, ' +
+                 'which is above the threshold of %(alertsCriticalHigh5xxRequests)s.') % this.config,
             },
           },
           {
-            alert: 'MicrosoftIISLowSuccessRateForWebsocketConnections',
+            alert: 'MicrosoftIISSuccessRateForWebsocket',
             expr: |||
               sum without (pid)  (increase(windows_iis_worker_websocket_connection_accepted_total[5m]) / clamp_min(increase(windows_iis_worker_websocket_connection_attempts_total[5m]),1)) * 100 > %(alertsCriticalLowWebsocketConnectionSuccessRate)s
```

Comment:
I know you didn't change this line but I think it has a bug:
- The code is calculating success percentage as: (accepted conns total / all conns attempts total) * 100
- Then we check if the success % is above a low success threshold 
- I think the comparison operator needs to be flipped: `<` instead of `>`

I keep finding these bugs in the existing code and I am going to add this as an example to https://github.com/grafana/cloud-onboarding/issues/9972.